Gemfile

```diff
@@ -157,7 +157,7 @@ group :test, :development do
   gem "factory_bot_rails", '6.2.0', require: false
   gem 'fuubar', '~> 2.5.1'
   gem 'json_spec', '~> 1.1.4'
-  gem 'knapsack_pro'
+  gem 'knapsack_pro', '7.0.0'
```

Comment:
We shouldn't restrict the version here. You might do that manually to update to a certain version but you don't need to commit this.